### PR TITLE
Partner page layout tweaks

### LINKED ISSUE #280: [Bug 2]: Partners page spacing does not match design
## Design notes
- Description needs much more spacing from the top with or without a logo appearing. 
- Where there’s no description or logo I think we need a note saying “Description coming soon” and lots of healthy spacing as per the design to keep the get in touch / address bits from jumping right to the top.
- Spacing where there’s no map, and no events, need spacing between “Check back for updates! And the bottom of the box.

## Acceptance Criteria TBC
- [ ] Description spacing from top matches design
- [ ] If no description, there is spacing instead
- [ ] If no map, nor events, need spacing between "Check back.." and bottom

### PR DESCRIPTION
**Pull request by ivankocienski**
Fixes #280

## Description
On partner show page

- default text when missing partner description
- padding above description

### CODE REVIEW

File: src/Page/Partners/Partner_.elm [hunk 1]
```diff
@@ -113,10 +113,19 @@ view maybeUrl sharedModel static =
     }
 
 
+descriptionText : Data.PlaceCal.Partners.Partner -> String
+descriptionText partner =
+    if String.isEmpty partner.description then
+        "Please ask " ++ partner.name ++ " for more information"
```
Comment (katjam) [inline src/Page/Partners/Partner_.elm:119]: This needs to come from `Copy.Text`. Also not sure if where the copy came from - but does it need a fullstop?

All of the interface text is meant to be centralised there. You'll have to make a new key that takes a String as a param.

Let me know if you get stuck.

**Review by katjam — CHANGES_REQUESTED** [review]
Please move copy into our translation system.

File: src/Copy/Text.elm [hunk 2]
```diff
@@ -341,3 +342,12 @@ chompTrailingUrlSlash urlString =
 urlToDisplay : String -> String
 urlToDisplay url =
     Url.fromString url |> urlRecombiner |> chompTrailingUrlSlash
+
+
+partnerDescriptionText : Partner -> String
```
Comment (katjam) [inline src/Copy/Text.elm:347]: Ah sorry. I didn't explain it very well. What you need to add is a `Copy.Key` that takes a `String` in the Partner page section - I guess `PartnerDescriptionEmptyText`?

See `PartnerUpcomingEventsText` as an example.

The url display stuff we did is a generic thing for all urls to display, but this is a specific bit of copy.

```

Comment (katjam) [reply src/Copy/Text.elm:347]: ALl of the copy comes from this translation function (`t`) or the APIs.

Comment (ivankocienski) [reply src/Copy/Text.elm:347]: That makes more sense

File: src/Copy/Text.elm [hunk 3]
```diff
@@ -220,6 +220,13 @@ t key =
         PartnerAddressEmptyText ->
             "No address provided"
 
+        PartnerDescriptionText partnerDescription partnerName ->
```
Comment (katjam) [inline src/Copy/Text.elm:223]: :tada:

**Review by katjam — APPROVED** [review]
Great!

**Pull request merged**